packages/design-system-react-native/src/components/Button.stories.tsx

```diff
@@ -18,8 +18,12 @@ export default ButtonMeta;
 export const Basic: StoryObj<typeof Button> = {
   render: () => {
     const tw = useTailwind();
+
     return (
       <View style={tw`bg-icon-default p-3`}>
+        <Text style={tw`text-display-md font-display-md text-primary-inverse`}>
+          Sample Text
+        </Text>
```

Comment:
Added simple addition to sample story to render changes before `Text` is merged

Reply:
Should this be rendering display-md style text?

<img width="532" alt="Screenshot 2024-11-26 at 9 15 21 AM" src="https://github.com/user-attachments/assets/787171a0-e1e6-40d9-b430-445ed706ce55">